tmva/sofie/README.md

```diff
@@ -44,4 +44,15 @@ To use the generated inference code:
 
 	#include "example_output.hxx"
 	float input[INPUT_SIZE];
```

Comment:
SOFIE supports other input formats, like RDataframe, should include that as well in the documentation

Reply:
Yes we could mention here and add a link to the corresponding tutorial

Reply:
Mentioned about RDataFrame inputs.